src/Engine/ProtoCore/DSASM/Executive.cs

```diff
@@ -2837,7 +2839,8 @@ private void ExecuteDebug(int exeblock, int entry, List<Instruction> breakpoints
                     runtimeCore.DebugProps.DebugEntryPC = pc;
                 }
 
-                core.Rmem = rmem;
+                //runtimeCore.RuntimeMemory = rmem;
+                runtimeCore.RuntimeMemory = rmem;
```

Comment:
In line 2570: `rmem = runtimeCore.RuntimeMemory;` and here `runtimeCore.RuntimeMemory =rmem`?